### PR TITLE
ENT-3110: Create Jenkins job for validating translation strings

### PR DESCRIPTION
**Pull request by sonalidhome**
- Added a job to validate translated strings
 - Updated the candlepinPipeline with newly added job
 - A shell script to call and execute candlepin cp-test script

### CODE REVIEW

**Review by nikosmoum — CHANGES_REQUESTED** [review]
We're in the middle of changing how the PR jenkins pipeline works (look at https://github.com/candlepin/candlepin/pull/2813 and https://github.com/candlepin/candlepin-jobs/pull/62).

The best course of action I think is to wait until both of those PRs are merged, and then re-work this PR to work with the new way of running jenkins jobs. (and by looking at those PRs, it looks like the new job will have to be defined in the candlepin repo, and not here)

**Comment by sonalidhome** [comment]
Closing this PR because of Jenkins pipeline changes in Candlepin. Created a PR https://github.com/candlepin/candlepin/pull/2847 for this job.

**Pull request closed without merging**
